src/utils/ZDTypes.ts

```diff
@@ -55,3 +56,8 @@ export type ZDTokensResponse = {
     previious_page: string
     count: number
 }
+
+export type StoredOauthUserToken = {
+    token: ClientOAuth2.Data,
+    is_agent: boolean
+}
```

Comment:
There's some inconsistency in this file for using commas, semicolons, or using no delimiter. 1/5 to pick one to be consistent

Reply:
changed all to remove commas and semicolons.